### DISCUSSION TITLE
get package github url

### DISCUSSION THREAD

**Discussion by bitboxer** [espanso/espanso#1206] [Feature requests and Ideas]
I would love to get the github url of a package so that I can go there and add a missing feature via a PR. Right now it is not possible for me to figure out where the code is. Maybe adding that to the plugin detail page on the hub would be great, too.

**Comment by federico-terzi** [comment]
Hey @bitboxer,

Thank you for the feedback! This seems very useful indeed. Given that I'm currently planning a major refactor of the package system, I'll make sure to consider this request when implementing it.

Cheers :)
